src/client/components/Buttons/Buttons.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```

Comment:
I think so You forgot to import(import "./Buttons.styles.css") css file in the js